core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -48,25 +49,12 @@
   }
 
   private static void formatFolder(String name, String folder) throws IOException {
-    UnderFileSystem ufs = UnderFileSystem.Factory.get(folder);
```

Comment:
@gpang  @peisun1115 , this is only called for formatting ramdisk dir structures. we only use localfs to access ramdisk